sigle/src/modules/editor/components/SlateEditor.tsx

```diff
@@ -594,52 +592,68 @@ export const SlateEditor = ({
   };
 
   return (
-    <React.Fragment>
-      <FixedContainer>
-        <AppBar>
-          <StyledAppBarRightContainer>
-            {loadingSave && (
-              <ButtonOutline style={{ marginRight: 6 }} disabled>
-                Saving ...
-              </ButtonOutline>
-            )}
-            {!loadingSave && story.type === 'public' && (
-              <ButtonOutline
-                style={{ marginRight: 6 }}
-                onClick={() => handleSave()}
-              >
+    <Container>
+      <Flex
+        as="header"
+        justify="between"
+        align="center"
+        css={{
+          mt: '$5',
+
+          '@md': {
+            mt: '$10',
+          },
+        }}
+      >
+        <Flex gap="10" align="center">
+          <Link href="/" passHref>
+            <a>
+              <ArrowLeftIcon />
+            </a>
+          </Link>
+          <Text size="sm">
+            <span style={{ fontWeight: 'bold' }}>{user?.username}</span>
```

Comment:
this is not using the stitches API but inlining the style, I think a better way to do this could be to use the Box component:

```suggestion
            <Box as="span" css={{ fontWeight: 'bold' }}>{user?.username}</Box>
```